iOS/DuckDuckGo/DuckDuckGoAlpha.entitlements

```diff
@@ -10,18 +10,16 @@
 	</array>
 	<key>com.apple.developer.siri</key>
 	<true/>
-	<key>com.apple.developer.web-browser</key>
```

Comment:
This custom entitlement was never officially available for the alpha build and now needs to be removed in order for code signing to work.